backend/src/main/java/com/redhat/cloud/notifications/routers/AdminService.java

```diff
@@ -31,15 +30,14 @@
 
     @GET
     @Produces(APPLICATION_JSON)
-    public Uni<Response> debugRbac(@QueryParam("rhid") String rhid) {
+    public Response debugRbac(@QueryParam("rhid") String rhid) {
 
-        Uni<RbacRaw> rbacRawUni = rbacServer.getRbacInfo("notifications,integrations", rhid);
-
-        return rbacRawUni
-                .onItem()
-                .transform(r -> Response.ok(r.data).build())
-                .onFailure()
-                .call(t -> Uni.createFrom().item(Response.serverError().entity("Rbac call failed -- see logs").build()));
+        try {
+            RbacRaw rbacRaw = rbacServer.getRbacInfo("notifications,integrations", rhid).await().indefinitely();
```

Comment:
`RbacServer` still has a reactive signature because it looks like `quarkus-security` can only be used with reactive code. As a consequence, the entire `com.redhat.cloud.notifications.auth` package is still reactive. Maybe I'll find a way to get rid of the reactiveness there later. It doesn't harm to keep it for now.

Reply:
How `indefinitely` is the `await().indefinitely()` ? we won't have requests waiting forever? Maybe we can set a reasonable time here.

Reply:
Good question. I'm not sure if `rbac-authentication/mp-rest/connectTimeout` and `rbac-authentication/mp-rest/readTimeout` would protect us against all possible issues or not. I'll add a timeout here to be safe.

Reply:
Done.